### ISSUE TITLE
29-Feb-2024 → D-MMMZ

### ISSUE THREAD

**Issue by JagadishQA0028** [gr2m/moment-parseformat#116]
`moment.parseFormat('29-Feb-2024')` parsed to `D-MMMZ`, I expected: DD-MMM-YYYY

**Comment by gr2m** [comment]
interesting. Does that only happen for Feb 29 or for any date?

**Comment by JagadishQA0028** [comment]
no for that specific format DD-MMM-YYYY I'm getting as D-MMMZ

**Comment by gr2m** [comment]
I haven't seen that format yet, is it common in your region?
I think it should work as expected if you leave out the dashes

**Comment by JagadishQA0028** [comment]
Yes, it is common for us, Our product needs this date format DD-MMM-YYYY (including dashes)
So to Verify the date is in correct format or not, I use moment.parseFormat('04-Mar-2024') but it is not working as expected.
